contrib/win32/win32compat/console.c

```diff
@@ -1,1624 +1,1630 @@
-/*
```

Comment:
hard to find what has changed.. could you please check why it is showing entire file as changed.

Reply:
Looks like VS2017 messed up indentation and line endings. I'll revert that commit and reissue the right one shortly.